pkg/apis/core/validation/validation.go

```diff
@@ -4194,12 +4197,20 @@ func ValidateNodeUpdate(node, oldNode *core.Node) field.ErrorList {
 		addresses[address] = true
 	}
 
-	if len(oldNode.Spec.PodCIDR) == 0 {
+	if len(oldNode.Spec.PodCIDRs) == 0 {
 		// Allow the controller manager to assign a CIDR to a node if it doesn't have one.
-		oldNode.Spec.PodCIDR = node.Spec.PodCIDR
+		//this is a no op for a string slice.
+		oldNode.Spec.PodCIDRs = node.Spec.PodCIDRs
 	} else {
-		if oldNode.Spec.PodCIDR != node.Spec.PodCIDR {
+		// compare the entire slice
+		if len(oldNode.Spec.PodCIDRs) != len(oldNode.Spec.PodCIDRs) {
```

Comment:
The condition on right of `!=` needs to be `node.Spec.PodCIDRs`

Reply:
Thanks, fixed with https://github.com/kubernetes/kubernetes/pull/73977/commits/51bcb16918f6396dfd5a61a0e2d936bdab399fd6